docs/reference/data_documentation_reference.rst

```diff
@@ -21,7 +21,7 @@ Here is an example of a site configuration from great_expectations.yml:
 .. code-block:: bash
 
     data_docs:
-      sites:
+      data_docs_sites
```

Comment:
wait, we're keeping this layer of nesting? I thought we were consolidating `data_docs` and `sites` into a single layer

Reply:
agreed -- this nesting should go away

Reply:
The nesting is gone - this one escaped my un-nesting campaign. Great catch!